contracts/GovernanceToken.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
-import "OpenZeppelin/openzeppelin-contracts-upgradeable@4.6.0/contracts/token/ERC20/ERC20Upgradeable.sol";
```

Comment:
I see the change to the import, but does this actually get used?

Reply:
My bad forgot to add it to contract definition. Should be ok now